project/FrgaalJavaCompiler.scala

```diff
@@ -9,19 +9,20 @@
  * (http://www.apache.org/licenses/LICENSE-2.0).
  */
 
-import sbt._
+import sbt.*
 import sbt.internal.inc.CompilerArguments
 import sbt.internal.inc.javac.JavacLogger
 import sbt.io.IO
 import sbt.util.Logger
-import xsbti.{PathBasedFile, Reporter, VirtualFile, Logger => XLogger}
-import xsbti.compile.{IncToolOptions, Output, JavaCompiler => XJavaCompiler}
+import xsbti.{PathBasedFile, Reporter, VirtualFile, Logger as XLogger}
```

Comment:
The usage of `as` breaks `scalafmtSbt`. Returning `=>` back fixes the formatting again.

Reply:
Fixed. Turns out my IDE thinks that this source file should be in Scala 3.